docs/resources/yaml.md.erb

```diff
@@ -11,36 +11,36 @@ Use the `yaml` InSpec audit resource to test configuration data in a Yaml file.
 
 ## Syntax
 
-A `yaml` resource block declares the configuration data to be tested. Assume the following Yaml file:
+A `yaml` resource block declares the configuration data to be tested. Assume the following Yaml file called 'filename.yml':
 
-    name: foo
-    array:
-      - zero
-      - one
+    name: myservice
+    hosts:
+      - ip1.host.com
+      - ip2.host.com
 
-This file can be queried using:
+This file can be queired using:
 
     describe yaml('filename.yml') do
-      its('name') { should eq 'foo' }
-      its(['array', 1]) { should eq 'one' }
+      its('name') { should eq 'myservice' }
+      its(['hosts', 0]) { should eq 'ip1.host.com' }
     end
 
 where
 
-* `name` is a configuration setting in a Yaml file
-* `should eq 'foo'` tests a value of `name` as read from a Yaml file versus the value declared in the test
+* `name` retrieves the 'myservice' value from file. That value is compared against the expressed expected value of 'myservice'.
+* `['hosts, 0']` retrieves the first element from the `hosts` array. That value is compared against the expressed expected value of 'ip1.host.com'.
 
 Like the `json` resource, the `yaml` resource can read a file, run a command, or accept content inline:
 
-    describe yaml('config.yaml') do
-      its(['driver', 'name']) { should eq 'vagrant' }
+    describe yaml('/etc/cloud/cloud.cfg') do
+      its('ssh_pwauth') { should eq 0 }
     end
 
     describe yaml({ command: 'retrieve_data.py --yaml' }) do
       its('state') { should eq 'open' }
     end
 
-    describe yaml({ content: \"key1: value1\nkey2: value2\" }) do
+    describe yaml({ content: "\"key1: value1\nkey2: value2\"" }) do
```

Comment:
Thanks!

Reply:
Really this and the next change are the ones I care about. I'll update the commit to include these two things.